front/src/common/api/osrdEditoastApi.ts

```diff
@@ -42,24 +42,101 @@ const osrdEditoastApi = generatedEditoastApi.enhanceEndpoints({
       // we don't want to invalidate the trainschedule tag here to prevent multiple calls
       invalidatesTags: ['timetable', 'scenarios'],
     },
-    // Invalidate the children count and last update timestamp
+
+    // Project handling
+    getProjects: {
+      providesTags: (result) => [
+        'projects',
```

Comment:
Do we need to keep the bare `projects` tag? I think the two tag types defined below should be enough?